### PR TITLE
add Duel.GetTossedCoinCount and Duel.GetTossedDiceCount

### PR DESCRIPTION
**Pull request by nekrozar**
Deal with the _Dueltainment_(https://github.com/Fluorohydride/ygopro-pre-script/blob/master/scripts/20AP-JP/c100213060.lua).

### CODE REVIEW

File: operations.cpp [hunk 1]
```diff
@@ -5051,6 +5051,7 @@ int32 field::toss_coin(uint16 step, effect * reason_effect, uint8 reason_player,
 			pduel->write_buffer8(count);
 			for(int32 i = 0; i < count; ++i) {
 				core.coin_result[i] = pduel->get_next_integer(0, 1);
+				core.tossed_dice_count[playerid]++;
```
Comment (mercury233) [inline operations.cpp]: i guess it should be `tossed_coin_count`

**Comment by DailyShana** [comment]
I think `EVENT_TOSS_DICE` and `EVENT_TOSS_COIN` are enough.

**Comment by nekrozar** [comment]
@DailyShana I tried `EVENT_TOSS_DICE` and `EVENT_TOSS_COIN`, but _Barrel Dragon_ get error message.

**Comment by DailyShana** [comment]
Since they are not used, I'll revert this.
and tossed count can be realized by global effects via `EVENT_TOSS_DICE` and `EVENT_TOSS_COIN`.

**Pull request merged**